### PR TITLE
fix: RTL mode arrow button direction

### PR DESCRIPTION
**Pull request by olshansky**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/1779
Related https://github.com/SAP/fundamental-ngx/issues/3674 

## Description
- Rotate arrow icon

### Before:
image

### After:
image

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Default:
Screen Shot 2020-11-06 at 11 06 01 AM

RTL:
Screen Shot 2020-11-06 at 11 06 10 AM

The Panel button does not "rotate" when we switch to RTL. It only changes in the RTL example but since we have a class for this button we can rotate it with styling

File: src/panel.scss [hunk 1]
```diff
@@ -53,10 +53,6 @@ $block: #{$fd-namespace}-panel;
 
   &__button {
     font-size: 1rem;
-
-    @include fd-rtl() {
-      @include fd-icon('slim-arrow-left');
```
Comment (InnaAtanasova) [inline src/panel.scss:58]: You can keep the `fd-rtl` mixin and just rotate the icon, not add a new one.

File: stories/panel/panel.stories.js [hunk 2]
```diff
@@ -197,3 +197,64 @@ fixedHeightContent.parameters = {
         iframeHeight: 300
     }
 };
+
+export const rtl = () => `
```
Comment (InnaAtanasova) [inline stories/panel/panel.stories.js:201]: Screen Shot 2020-11-06 at 11 14 23 AM

You don't have to add this example, we have a "switcher" on the top of the doc page. All examples should apply the RTL rules when you switch to this mode.

File: src/panel.scss (hunk 1 above)
Comment (olshansky) [reply src/panel.scss:58]: @InnaAtanasova thanks, fixed

**Review by InnaAtanasova — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Review by JKMarkowski — APPROVED** [review]

**Comment by JKMarkowski** [comment]
@olshansky Can you push some empty commit to this branch, just to trigger chromatic pipeline?

**Pull request merged**